src/graphql/resolvers.js

```diff
@@ -1,17 +1,44 @@
 import {
   getContributorInfo,
+  getOpenIssuesByLabel,
   isGitHubContributor,
   inviteIfNecessary
 } from '../lib/github';
-import { createShopifyCustomer } from '../lib/shopify';
+import { createShopifyCustomer as createShopifyCustomerRest } from '../lib/shopify';
+import { createShopifyCustomer } from '../lib/shopify-graphql';
 import getLogger from '../lib/logger';
+import { prisma } from '../prisma-client';
 
 const logger = getLogger('graphql/resolvers');
 
 export default {
   Query: {
     contributorInformation: async (_, { githubUsername }) => {
       return await getContributorInfo(githubUsername);
+    },
+    openIssues: async (_, { label }) => {
+      return await getOpenIssuesByLabel(label);
+    },
+    getContributor: async (_, { githubUsername }) => {
+      // TODO when should we create the contributor the first time?
```

Comment:
On form submit. If a contributor doesn't exist, they should create an account on the form submit to get the code. Other than that, all we need is the github info in the UI.